students/MikeShand/lesson03/strformat_lab.py

```diff
@@ -5,4 +5,49 @@
 
 x=(2,123.467,10000,12345.67)
 
-'file{:0>4d}: {:3.2f},{:.2e},{:03.2e}'.format(x[0],x[1],x[2],x[3])
+'file{:0>4d}: {:3.2f},{:.2e},{:03.2e}'.format(*x)
+
+
+#Task 2
+
+'file{:-6d}: {:3.0f},{:10d},{:^6f}'.format(*x)

```

Comment:
looks like you're missing `print()` function calls in couple places

Reply:
for task 2 you should try to use different type of formatting, you can use keyword style:

```
>>> print("hello {var}".format(var="world"))
hello world
>>> var = "world"

```

or f-strings:
```
>>> print(f"hello {var}")
hello world
```